src/components/control/date.jsx

```diff
@@ -165,6 +190,7 @@ const Date = forwardRef(function Date(props, forwardedRef) {
         name={props.name}
         onChange={onInputChange}
         onClick={onInputClick}
+        onFocus={onFocusIntoDateControl}
```

Comment:
I wonder if we can just hoist the `onFocus` handler to the root container since we really care about _any_ focus within.

Reply:
Yeah pulling this up to the container div works. I'm going to do that